app/views/main.scala.html

```diff
@@ -5,9 +5,9 @@
 <html>
     <head>
         <title>@title</title>
-        <link rel="stylesheet" media="screen" href="@routes.Assets.at("stylesheets/main.css")">
-        <link rel="shortcut icon" type="image/png" href="@routes.Assets.at("images/favicon.png")">
-        <script src="@routes.Assets.at("javascripts/jquery-1.9.0.min.js")" type="text/javascript"></script>
+@*        <link rel="stylesheet" media="screen" href="@routes.Assets.at("stylesheets/main.css")">*@
```

Comment:
can these be removed rather than commented out? Or maybe they need adding back

Reply:
Looks like this is something we commented out when we were first trying to make it compile so should be added back in. I'll try uncommenting it and see what happens...

Reply:
Don't forget that otherwise https://prism.gutools.co.uk/ will look ugly. Oh wait.

Reply:
✅